src/SFA.DAS.ApprenticeCommitments.Web.UnitTests/HashedIdTests.cs

```diff
@@ -1,30 +1,46 @@
-﻿using AutoFixture.NUnit3;
+﻿using System.Collections.Generic;
+using AutoFixture.NUnit3;
 using FluentAssertions;
 using NUnit.Framework;
 using SFA.DAS.ApprenticeCommitments.Web.Identity;
-using SFA.DAS.HashingService;
+using SFA.DAS.Encoding;
 using static SFA.DAS.ApprenticeCommitments.Web.UnitTests.EqualityTests;
 
 namespace SFA.DAS.ApprenticeCommitments.Web.UnitTests
 {
     public class HashedIdTests
     {
         private const string alphabet = "abcdefgh12345678";
-        public static IHashingService Hashing = new HashingService.HashingService(alphabet, "testing");
+
+        public static IEncodingService Hashing = new EncodingService(new EncodingConfig
+        {
+            Encodings = new List<Encoding.Encoding>
+            {
+                new Encoding.Encoding
+                {
+                    Alphabet = alphabet, 
+                    EncodingType = "ApprenticeshipId",
+                    MinHashLength = 6, 
+                    Salt = "Salt"
+                }
+            }
+        });
 
         public static HashedId NewHashedId(string value)
             => HashedId.Create(value, Hashing);
 
         public static HashedId NewHashedId(long value)
-            => HashedId.Create(Hashing.HashValue(value), Hashing);
+            => HashedId.Create(Hashing.Encode(value, EncodingType.ApprenticeshipId), Hashing);
 
-        [TestCase(1, "42g87g")]
+        [TestCase(1, "4b8bg8")]
```

Comment:
Should the resulting hash values not end up the same as when we used the other hashing library?

Reply:
No, it's because each encoding type can use different alphabets and salts. I think the logic is an external party can't use one hashed value type as a substitute (bit over the top for me)